src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -693,7 +693,6 @@ internal async Task<Result> GetMethodLocation(MessageId id, JObject args, Cancel
 
             // GetAssemblyByName seems to work on file names
             AssemblyInfo assembly = store.GetAssemblyByName(aname);
-            assembly ??= store.GetAssemblyByName(aname + ".exe");
```

Comment:
Why we decided to ignore execs? This does not seem connected with JMC setting.

Reply:
Because GetAssemblyByName is not considering the extension anymore. And probably also the extension in the future will probably be only .wasm because of the webcil work.